contributors/guide/github-workflow.md

```diff
@@ -17,23 +17,14 @@ description: |
 
 ## 2. Clone fork to local storage
 
-Per Go's [workspace instructions][go-workspace], place Kubernetes' code on your
-`GOPATH` using the following cloning procedure.
+In your shell, define a local working directory as `working_dir`. As [GO Modules] got introduced and became default in GO 1.13, you don't have to put your Go project under `GOPATH`.
 
-[go-workspace]: https://golang.org/doc/code.html#Workspaces
-
-In your shell, define a local working directory as `working_dir`. If your `GOPATH` has multiple paths, pick
-just one and use it instead of `$GOPATH`. You must follow exactly this pattern,
-neither `$GOPATH/src/github.com/${your github profile name}/`
-nor any other pattern will work.
+[GO Modules]: https://go.dev/blog/using-go-modules
 
 ```sh
-export working_dir="$(go env GOPATH)/src/k8s.io"
+export working_dir="${HOME}" # Or any directory you want
```

Comment:
I don't think `HOME` is a good default to show here. Where they will want to have their source will vary from person to person, but placing it directly in `$HOME` would not be a reasonable default.

Maybe 

```sh
export working_dir="${HOME}/src/k8s.io" # Change to your preferred location for source code
```

Reply:
SGTM, updated.